ui/cypress/e2e/tasks.test.ts

```diff
@@ -61,80 +61,69 @@ from(bucket: "${name}")
       .and('contain', taskName)
   })
 
-  it('can delete a task', () => {
-    cy.get('@org').then(({id}: Organization) => {
-      cy.get<string>('@token').then(token => {
-        cy.createTask(token, id)
-        cy.createTask(token, id)
-      })
-
-      cy.fixture('routes').then(({orgs}) => {
-        cy.visit(`${orgs}/${id}/tasks`)
-      })
-
-      cy.getByTestID('task-card').should('have.length', 2)
-
-      cy.getByTestID('task-card')
-        .first()
-        .trigger('mouseover')
-        .within(() => {
-          cy.getByTestID('context-delete-menu').click()
-          cy.getByTestID('context-delete-task').click()
+  describe('When tasks already exist', () => {
+    beforeEach(() => {
+      cy.get('@org').then(({id}: Organization) => {
+        cy.get<string>('@token').then(token => {
+          cy.createTask(token, id)
         })
-
-      cy.getByTestID('task-card').should('have.length', 1)
-    })
-  })
-
-  it('can disable a task', () => {
-    cy.get('@org').then(({id}: Organization) => {
-      cy.get<string>('@token').then(token => {
-        cy.createTask(token, id)
       })
     })
 
-    cy.getByTestID('task-card--slide-toggle').should('have.class', 'active')
+    it('can edit a task', () => {
+      // Disabling the test
+      cy.getByTestID('task-card--slide-toggle').should('have.class', 'active')
 
-    cy.getByTestID('task-card--slide-toggle').click()
+      cy.getByTestID('task-card--slide-toggle').click()
 
-    cy.getByTestID('task-card--slide-toggle').should('not.have.class', 'active')
-  })
+      cy.getByTestID('task-card--slide-toggle').should(
+        'not.have.class',
+        'active'
+      )
 
-  it('can edit a tasks name', () => {
-    cy.get('@org').then(({id}: Organization) => {
-      cy.get<string>('@token').then(token => {
-        cy.createTask(token, id)
-      })
-    })
+      // Editing a name
+      const newName = 'Task'
 
-    const newName = 'Task'
+      cy.getByTestID('task-card').within(() => {
+        cy.getByTestID('task-card--name').trigger('mouseover')
 
-    cy.getByTestID('task-card').within(() => {
-      cy.getByTestID('task-card--name').trigger('mouseover')
+        cy.getByTestID('task-card--name-button').click()
 
-      cy.getByTestID('task-card--name-button').click()
+        cy.get('.cf-input-field')
+          .type(newName)
+          .type('{enter}')
+      })
 
-      cy.get('.cf-input-field')
-        .type(newName)
-        .type('{enter}')
+      cy.getByTestID('notification-success').should('exist')
+      cy.getByTestID('task-card').should('contain', newName)
     })
 
-    cy.fixture('routes').then(({orgs}) => {
-      cy.get('@org').then(({id}: Organization) => {
-        cy.visit(`${orgs}/${id}/tasks`)
-      })
+    it.only('can delete a task', () => {
```

Comment:
`.only`?

Reply:
hot tip lads and lassies: `yarn prettier:fix` doesn't fix `it.only`.

Reply:
🤣